src/pages/steps/CalendarStep/Days.tsx

```diff
@@ -102,7 +102,6 @@ export const Days = ({
               <DatePeriodPicker
                 spacing={3}
                 id={`calendar-step-day-${day.id}`}
-                minDate={new Date()}
```

Comment:
What is the default date when you enter a new event?

Reply:
Still the current date so should be similar in behavior